dependencies.gradle

```diff
@@ -5,45 +5,123 @@ ext.versions = [
         'targetSdk'         : 30,
         'sourceCompat'      : JavaVersion.VERSION_11,
         'targetCompat'      : JavaVersion.VERSION_11,
+]
 
-        'kotlin'           : '1.5.30',
-        'kotlinCoroutines' : '1.5.1',
-        'dagger'           : '2.38.1',
-        'timber'           : '5.0.1',
-        'work'             : '2.5.0',
-        'lifecycle'        : '2.2.0',
-        'moshi'            : '1.12.0',
-        'arch'             : '2.1.0',
-        'markwon'          : '4.6.2',
-        'arrow'            : '0.8.2',
-        'rxKotlin'         : '2.4.0',
-        'rxAndroid'        : '2.1.1',
+// Ref: https://kotlinlang.org/releases.html
```

Comment:
This ref is use to ensure kotlin and kotlinCoroutines version are aligned, so can you move to the line below please?

Reply:
We will do that later, let see what happen for dependabot current PRs when this one will be merged